tendrl/commons/message.py

```diff
@@ -153,7 +153,7 @@ def serialize_message(obj):
         serial = obj.isoformat()
         return serial
     elif isinstance(obj, Exception):
-        return yaml.safe_dump(obj)
+        return yaml.safe_dump(obj.message)
```

Comment:
You can view it @ https://github.com/Tendrl/alerting/pull/41/files#diff-832c56a6c92c3a3783467abf5cef02c1R105